Miscellaneous code cleanup

basil opened the pull request:
CC @simschla

simschla reviewed (approved):
According to which formatting style did you rewrite the imports? (just for my future reference)

basil commented:
No specific style: I just deleted unused imports and replaced deprecated imports with non-deprecated equivalents. Since some of the star imports were deprecated, I also replaced all star imports with specific imports so that I could migrate to the non-deprecated equivalent.
